src/env/default.rs

```diff
@@ -257,12 +277,113 @@ impl WriteExt for LogFile {
     }
 }
 
+pub struct AioContext {
+    inner: Option<Arc<LogFd>>,
+    index: usize,
+    aio_vec: Vec<aiocb>,
+    pub(crate) buf_vec: Vec<Vec<u8>>,
+}
+impl AioContext {
+    pub fn new(block_sum: usize) -> Self {
+        let mut aio_vec = vec![];
+        let buf_vec = vec![];
+        unsafe {
+            for _ in 0..block_sum {
+                aio_vec.push(mem::zeroed::<libc::aiocb>());
+            }
```

Comment:
Why creating them before hand, instead of creating them the same time as `buf_vec`?

Reply:
@tabokie If `libc:: aiocb` is created in multiple function calls, the compiler will assign the same address to the pointer every time, which will leads incorrect result. I'm not so familiar with the memory allocate API of trust, do you have any other good methods?

Reply:
I don't think it's possible to have multiple variables with the same address. It's possible that you mistakenly free some of them.